packages/jspsych/src/modules/plugins.ts

```diff
@@ -145,7 +145,11 @@ export interface PluginInfo {
 }
 
 export interface JsPsychPlugin<I extends PluginInfo> {
-  trial(display_element: HTMLElement, trial: TrialType<I>): void;
+  trial(
+    display_element: HTMLElement,
+    trial: TrialType<I>,
+    on_load?: () => void
```

Comment:
```suggestion
    on_load: () => void
```
I think it is always provided, so this would reflect that

Reply:
My thought was that a plugin could choose whether or not to receive it, so that we don't have to update plugins that aren't going to explicitly call it for now. It will still be provided of course.

Reply:
Sorry, my bad! I was overlooking that this is an interface definition and expecting that the provided function might be typed optional in plugins, which it obviously doesn't.